scripts/build/builders/openiotsdk.py

```diff
@@ -58,6 +59,10 @@ def ExamplePath(self):
         return os.path.join(self.root, 'examples', self.app.ExampleName, 'openiotsdk')
 
     def generate(self):
+        # Remove access to ARM GCC toolchain from Matter environment, use higher version from OIS environment
+        cmd = "PATH=$(echo $PATH | sed 's/:/\n/g' | grep -v $PW_ARM_CIPD_INSTALL_DIR | xargs | tr ' ' ':')"
+        self._Execute(['bash', '-c', cmd],
```

Comment:
if you execute `bash -c` would the path changes not only persist in the current shell invocation (i.e. CMAKE will not see it)?

You may need to execute the entire PATH + cmake into one large bash command ... I believe this is what we do for idf.py builds.

Also wondering if this path change is needed during the build phase or if cmake caches compiler paths.

Reply:
I just noticed that it won't be needed anymore because the Pigweed environment now has the correct version of GCC ARM compiler (10.3-2021.10).